src/scala/scripts/ScalaPBGenerator.scala

```diff
@@ -39,20 +51,19 @@ class ScalaPBGenerator extends Processor {
 
   def processRequest(args: java.util.List[String]) {
     val extractRequestResult = PBGenerateRequest.from(args)
+    setupIncludedProto(extractRequestResult.includedProto)
+
     val config = ScalaPBC.processArgs(extractRequestResult.scalaPBArgs.toArray)
     val code = ProtocBridge.runWithGenerators(
       protoc = a => com.github.os72.protocjar.Protoc.runProtoc(a.toArray),
       namedGenerators = Seq("scala" -> ScalaPbCodeGenerator),
       params = config.args)
 
     try {
-      if (!config.throwException) {
```

Comment:
The previous if/else if caused issues where i was having failures but it just produced empty outputs rather than quitting properly.

I believe this configuration handling is an odd spill over from using the ScalaPB arg parsing. It looks like the behavior in scalapb is to let the user use the exit code for error handling or throw a java exception. And it defaults to throwException in the config as false. 

https://github.com/scalapb/ScalaPB/blob/master/scalapbc/src/main/scala/scalapb/ScalaPBC.scala#L67-L72

I don't think we have any use/interest in this config variable unless i'm missing something? since the user doesn't have any direct control/utility for changing how the error handling is done. (right now if `config.throwException` is false, the default, we will always build the jar(though its probably empty), and not throw the error from my read of it).